README.md

```diff
@@ -119,6 +124,162 @@ resource provided above.
     $ git remote add upstream https://github.com/shescoding/shescoding-dot-org.git
     ```
 
++ To complete installation, skip past the directions for Windows 10 [here](#mac,-ubuntu,-and-windows)
+---
+---
+---
+  #### Windows 10 Users: Using the Ubuntu: App
+
+  Go to [Windows App Store](https://www.microsoft.com/store/productId/9PJN388HP8C9) and download the most stable version of Ubuntu, then
+
+    + Go to developer settings by searching "Developer"
+
+    + In the results under *Settings*, click  *For developer settings*
+
+    + Select *Developer Mode*
+
+  Open the Ubuntu app and complete setup (username, password, etc.), then run:
+
```

Comment:
this section doesn't seem to be formatted correctly 
![styling](https://user-images.githubusercontent.com/18508181/50018519-734d1e80-ff84-11e8-95c1-8c09f2ac6e81.JPG)

Reply:
This has been fixed